source/extensions/filters/http/transformation/transformation_filter_config.cc

```diff
@@ -202,6 +214,7 @@ void PerStageRouteTransformationFilterConfig::addTransformation(
       TransformerPairConstSharedPtr transformer_pair =
           std::make_unique<TransformerPair>(request_transformation,
                                             response_transformation,
+                                            nullptr,
```

Comment:
Should access log transformations also be included in StageRouteTransformationFilterConfig? Didn't seem like it when looking at the code but I'm not familiar with eh distinction between this and regular TransformationFIlterConfig.

Reply:
the staged ones are new / supported, the old ones are deprecated. stages allow us to run transformations more than once at different times in the filter chain

Reply:
I'm not sure if DLP supports stages today, the most relevant API for this feature will be that one.

Reply:
Right, I was thinking this was the open source transformation filter when I wrote that comment, not DLP 🤦